webrtc.html

```diff
@@ -1910,8 +1908,8 @@
 
         <p>An <code><a>RTCPeerConnection</a></code> object MUST not be garbage
         collected as long as any event can cause an event handler to be
-        triggered on the object. When the object's
-        <a>signaling state</a> is <code>closed</code>, no such event handler can be
+        triggered on the object. When the object's [[<a>isClosed</a>]]
+        internal slot is <code>true</code>, no such event handler can be
```

Comment:
Not a problem with this patch, but this reminds me this section on garbage collection is quite wrong. All closed peer connections are not "safe to garbage collect", as this sentence runs on saying. Nevermind any outstanding JS references, but I thought we agreed one can call `getStats` on closed peer connections as well. Is this section even needed, or is being derived from `EventTarget` perhaps sufficient (I don't know, just guessing it might be)?